server/controllers/submission.js

```diff
@@ -0,0 +1,31 @@
+const Submission = require("../models/Submission");
+const asyncHandler = require("express-async-handler");
+
+exports.createSubmission = asyncHandler(async (req, res) => {
+  const body = req.body;
+  try {
+    //checking if the user for the contest, has already submitted
+    const previousSubmission = await Submission.findOne({
+      contest: req.params.id,
+      user: req.user.id,
+    });
+
+    if (previousSubmission) {
+      previousSubmission.storeUrl = previousSubmission.storeUrl.concat(
+        body.storeUrl
+      );
+      await previousSubmission.save();
+      res.status(201).json(previousSubmission);
```

Comment:
Please use status code 200 here. Code 201 is for successful resource creation :)

Reply:
oh yeah. thanks